addons/microsoft_calendar/models/calendar_recurrence_rule.py

```diff
@@ -121,37 +134,29 @@ def _get_microsoft_sync_domain(self):
         # We also prevent sync of other user recurrent events.
         return [('calendar_event_ids.user_id', '=', self.env.user.id), ('rrule', '!=', False)]
 
-
     def _cancel_microsoft(self):
         self.calendar_event_ids._cancel_microsoft()
         super()._cancel_microsoft()
 
     @api.model
-    def _microsoft_to_odoo_values(self, microsoft_recurrence, default_reminders=(), default_values={}):
+    def _microsoft_to_odoo_values(self, microsoft_recurrence, default_values=None, with_ids=False):
         recurrence = microsoft_recurrence.get_recurrence()
 
-        return {
-            **recurrence,
-            'microsoft_id': microsoft_recurrence.id,
-        }
+        if with_ids:
+            recurrence = {
+                **recurrence,
+                'ms_organizer_event_id': microsoft_recurrence.id,
+                'ms_universal_event_id': microsoft_recurrence.iCalUId,
+            }
```

Comment:
Normally, we should not change the signature in stable version. 
If we provide new parameter with default values, it's less problematic. Could you keep the default_reminders  parameter even if you don't use it and add a comment to remove it in master?